<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Import Tool</string>
  </property>
  <widget class="QListWidget" name="listWidget_files">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>500</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_files">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>471</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>The selected files and folders will be imported. Please uncheck the files and folders that you do NOT want imported.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_nickname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nickname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter a nickname for this deposition.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>490</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>490</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Selected Files</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
